engine/api/application/gorp_model.go

```diff
@@ -0,0 +1,57 @@
+package application
+
+import (
+	"encoding/json"
+
+	"github.com/go-gorp/gorp"
+	"github.com/ovh/cds/engine/api/database/gorpmapping"
+	"github.com/ovh/cds/sdk"
+)
+
+type dbApplication sdk.Application
+type dbVariable sdk.Variable
+
+func init() {
+	gorpmapping.Register(gorpmapping.New(dbApplication{}, "application", true, "id"))
+}
+
+// PostGet is a db hook
+func (a *dbApplication) PostGet(db gorp.SqlExecutor) error {
+	metadataStr, err := db.SelectNullStr("select metadata from application where id = $1", a.ID)
+	if err != nil {
+		return err
+	}
+
+	if metadataStr.Valid {
+		metadata := sdk.Metadata{}
+		if err := json.Unmarshal([]byte(metadataStr.String), &metadata); err != nil {
+			return err
+		}
+		a.Metadata = metadata
+	}
+
+	pkey, errP := db.SelectStr("select projectkey from project where id = $1", a.ProjectID)
```

Comment:
humm not so easy. Try different things without any success. 
Last chance : add it in application table... what do you think about it ?